src/lib/UniversalDiscovery/Event/Subscriber/ObjectRelationAllowedContentTypes.php

```diff
@@ -43,10 +43,10 @@ public function onUdwConfigResolve(ConfigResolveEvent $event): void
         }
 
         $config['content_on_the_fly']['allowed_content_types'] = array_unique(
```

Comment:
`array_unique` is no longer needed here if we use `array_intersect`.

Reply:
removed